client/my-sites/theme/main.jsx

```diff
@@ -498,17 +498,17 @@ const ThemeSheet = React.createClass( {
 
 		return (
 			<Main className="theme__sheet">
-				<QueryCanonicalTheme themeId={ this.props.id } siteId={ siteID } />
+				<QueryCanonicalTheme themeId={ this.props.id } siteId={ siteId } />
 				{ currentUserId && <QueryUserPurchases userId={ currentUserId } /> }
-				{ siteID && <QuerySitePurchases siteId={ siteID } /> }
-				{ siteID && <QuerySitePlans siteId={ siteID } /> }
+				{ siteId && <QuerySitePurchases siteId={ siteId } /> }
+				{ siteId && <QuerySitePlans siteId={ siteId } /> }
```

Comment:
QuerySitePlans can handle a falsey siteId, so this is unnecessary to check here.